README.md

```diff
@@ -0,0 +1,13 @@
+# testium-cookie
+
+WebDriver does not return status codes or response headers.
+This module stores those in a special cookie that can be read
+from normal WebDriver methods.
+
+## modifyResponse(res)
+
+Modifies an existing node http response to inject the `_testium_` cookie.
+
+## parseTestiumCookie(cookies)
```

Comment:
Should this be `getTestiumCookie`?  Additionally, I don't see `parseTestiumCookie` being exported.